### PR TITLE
chore: fix all solhint issues

### PR DESCRIPTION
**Pull request by filoozom**
Uses named imports instead of global ones. Also fixes one tiny linting issue.

### CODE REVIEW

File: contracts/SBCDepositContract.sol [hunk 1]
```diff
@@ -264,12 +265,11 @@ contract SBCDepositContract is
      * NOTE: This function signature is hardcoded in the Gnosis execution layer clients. Changing this signature without updating the
      * clients will cause block verification of any post-shangai block to fail. The function signature cannonical spec is here
      * https://github.com/gnosischain/specs/blob/master/execution/withdrawals.md
-     * @param _deprecatedUnused Previously `maxFailedWithdrawalsToProcess` currently deprecated and ignored
      * @param _amounts Array of amounts to be withdrawn.
      * @param _addresses Array of addresses that should receive the corresponding amount of tokens.
      */
     function executeSystemWithdrawals(
-        uint256 _deprecatedUnused,
+        uint256 /* _deprecatedUnused */,
```
Comment (dapplion) [inline contracts/SBCDepositContract.sol:272]: Unrelated diff, please revert lines 267 and 272

**Review by dapplion — CHANGES_REQUESTED** [review]

File: contracts/SBCDepositContract.sol (hunk 1 above)
Comment (filoozom) [reply contracts/SBCDepositContract.sol:272]: Well, maybe I should rename the PR to "fix all solhint issues", which this does. Or do you still prefer reverting that?

Comment (dapplion) [reply contracts/SBCDepositContract.sol:272]: Oh if solhint requires that change then all good :+1:

File: contracts/SBCDepositContract.sol [hunk 2]
```diff
@@ -264,12 +265,11 @@ contract SBCDepositContract is
      * NOTE: This function signature is hardcoded in the Gnosis execution layer clients. Changing this signature without updating the
      * clients will cause block verification of any post-shangai block to fail. The function signature cannonical spec is here
      * https://github.com/gnosischain/specs/blob/master/execution/withdrawals.md
-     * @param _deprecatedUnused Previously `maxFailedWithdrawalsToProcess` currently deprecated and ignored
```
Comment (dapplion) [inline contracts/SBCDepositContract.sol:267]: Does solhint require to remove this line? If no, please keep. If yes, then convert to non param line we need the context of that that first variable refers to

File: contracts/SBCDepositContract.sol (hunk 1 above)
Comment (filoozom) [reply contracts/SBCDepositContract.sol:272]: It does:
```
,Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
   --> project:/contracts/SBCDepositContract.sol:273:9:
    |
273 |         uint256 _deprecatedUnused,
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Review by 4rgon4ut — APPROVED** [review]
Good job!

**Review by dapplion — APPROVED** [review]
